libgphoto2/gphoto2-setting.c

```diff
@@ -37,7 +37,7 @@
 #ifdef WIN32
 /* Win32 headers may use interface as a define; temporarily correct this */
 #define interface struct
-#include <Shlobj.h>
```

Comment:
On Windows the case doesn't matter, but it is not the case on Linux. It seems the mingw provides the header in the lowercase (at least on Fedora 33), so use the lowercase.

Reply:
Looks obvious.